webpack/components/ProxmoxContainer/MountPoint.js

```diff
@@ -0,0 +1,66 @@
+import React, { useState, useEffect } from 'react';
+import { Divider } from '@patternfly/react-core';
+import InputField from '../common/FormInputs';
+import ProxmoxComputeSelectors from '../ProxmoxComputeSelectors';
+import { translate as __ } from 'foremanReact/common/I18n';
+
+const MountPoint = ({ id, data, storagesMap }) => {
+  const [mp, setMp] = useState(data);
+  const handleChange = e => {
+    const { name, value } = e.target;
+    const updatedKey = Object.keys(mp).find(key => mp[key].name === name);
+    const updatedData = {
+      ...mp,
+      [updatedKey]: { ...mp[updatedKey], value },
+    };
+    setMp(updatedData);
+  };
+
+  return (
+    <div>
+      <Divider component="li" style={{ marginBottom: '2rem' }} />
+      <InputField
+        name={mp.storage.name}
+        label={__('Storage')}
+        type="select"
+        options={storagesMap}
+        value={mp.storage.value}
+        onChange={handleChange}
+      />
+      <InputField
```

Comment:
All the other InputFields have type set, but this does not. Is it on purpose?

Reply:
because default type is "text" since most of the fields are text.